backend/tests/tests.py

```diff
@@ -0,0 +1,32 @@
+import unittest
+import os
+import sys
+
+def run_tests():
+    """
+    Discover and run all tests in the 'tests' directory.
+    """
+    # Get the directory where this script is located
+    start_dir = os.path.dirname(os.path.abspath(__file__))
+    
+    # Add the src directory to sys.path
+    src_dir = os.path.abspath(os.path.join(start_dir, '../src'))
+    if src_dir not in sys.path:
+        sys.path.insert(0, src_dir)
+    
+    # Create a test loader
+    loader = unittest.TestLoader()
```

Comment:
I've never seen someone use `unittest` before lol. Do you want to try using the [pytest](https://docs.pytest.org/en/stable/) library instead? It obviates the need for the boilerplate that you wrote.